.github/workflows/release.yml

```diff
@@ -394,8 +402,9 @@ jobs:
       - name: Build and install dependencies
         run: |
           yum update -y
-          yum install -y zlib-devel zlib-static
+          yum install -y zlib-devel zlib-static cmake devtoolset-8 wget unzip
```

Comment:
We have devtoolset-11 on x86_64 and devtoolset-10 on aarch64. Do we still need devtoolset-8?